src/HeaderUserInformation/HeaderUserInformation.tsx

```diff
@@ -62,7 +70,14 @@ const HeaderUserInformation: FC<HeaderUserInformationProps> = ({
   }
 
   return (
-    <WrapperDiv onClick={onClick} id={id}>
+    <WrapperDiv
+      onClick={onClick}
+      id={id}
+      role={role}
+      aria-haspopup={popUpType}
+      {...(popUpId ? { 'aria-controls': popUpId } : {})}
```

Comment:
You can simplify it to: `aria-controls={popUpId}` since if the popUpId is undefined it is equivalent for the prop.